backend/functions/lib/adapters/gateways/cases.cosmosdb.repository.ts

```diff
@@ -96,9 +96,11 @@ export class CasesCosmosDbRepository implements CasesRepository {
           .database(this.cosmosConfig.databaseName)
           .container(this.containerName)
           .items.create(itemToCreate);
+        context.logger.info(MODULE_NAME, 'Created the following resource:', resource);
```

Comment:
Don't agree with logging out this whole object.

Reply:
Agreed

Reply:
Second

Reply:
As implemented, we can't know what to log here because resource is a generic. We either need to not log in here at all or instead extend our sanitize functionality to remove PII.

Reply:
We added functionality to remove any properties that are disallowed. The objects being logged here are generics, so we don't know what properties they have and which if any are inappropriate for logging.